docs/source/en/_toctree.yml

```diff
@@ -323,6 +323,8 @@
         title: NLLB
       - local: model_doc/nystromformer
         title: Nyströmformer
+      - local: model_doc/oneformer
+        title: OneFormer
```

Comment:
You can move OneFormer to the "multimodal" section, as it leverages both image + text modalities.